tests/test_grafana.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for the Grafana containers."""
+
+import pytest
+import requests
+import tenacity
+from pytest_container.container import ContainerData
+
+from bci_tester.data import GRAFANA_CONTAINERS
+
+
+@pytest.mark.parametrize("container", GRAFANA_CONTAINERS, indirect=True)
+def test_prometheus_healthy(container: ContainerData) -> None:
+    """Simple smoke test verifying that Grafana is healthy."""
+
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(5), wait=tenacity.wait_exponential()
+    )
+    def _fetch_grafana_health() -> requests.Response:
+        port = container.forwarded_ports[0].host_port
+        return requests.get(f"http://localhost:{port}/api/health", timeout=2)
+
+    resp = _fetch_grafana_health()
+    baseurl = container.container.baseurl
+    assert baseurl
+    assert resp.status_code == 200
```

Comment:
JFYI, you can also do this:
```suggestion
    resp.raise_for_status()
```